src/dialogs.c

```diff
@@ -82,9 +82,12 @@ void oregano_error_with_title (gchar *title, gchar *msg)
 	// make sure that this is running in the main thread
 	if (tid && (getpid() != tid)) {
 		OreganoTitleMsg *tm = g_malloc (sizeof (OreganoTitleMsg));
-		tm->title = g_strdup (title);
-		tm->msg = g_strdup (msg);
-		g_idle_add_full (G_PRIORITY_DEFAULT_IDLE, (GSourceFunc) oregano_schedule_error_with_title, tm, NULL);
+		if (G_LIKELY (tm)) {
```

Comment:
If this fails we are doomed anyways, please change into assert/error. This is unrecoverable.

Reply:
I have just followed the current coding style, so that it is coherent with the rest of the existing code...

Reply:
I have amended it now. And the other similar existing code...